.travis.yml

```diff
@@ -23,12 +23,16 @@ script:
   - go test -v ./...
   - goveralls -service=travis-ci
 
+matrix:
+  fast_finish: true
+  allow_failures:
+    - go: master
```

Comment:
why even build with go master then? Why don't we just remove it from the list of versions. 

We should be able to add in any newer versions of the language once they have been officially released.

Reply:
OK, I removed it at e135572